Gemfile

```diff
@@ -51,6 +51,7 @@ group :development, :test do
   gem 'factory_girl_rails'
   gem 'database_cleaner'
   gem 'rails-controller-testing'
+  gem 'capybara'
```

Comment:
We'll probably want to swap this for rspec as that includes the capybara as well as the ability todo unit and functional tests i.e. https://stackoverflow.com/questions/22491145/difference-between-rspec-capybara-and-cucumber

Reply:
If I remove the Capybara dependency, I get the following error:
```
Failure/Error: visit '/'
     
     RuntimeError:
       Capybara not loaded, please add it to your Gemfile:
     
       gem "capybara"
```
I think maybe because I am using the Capybara DSL?

Reply:
Seems like the capybara gem needs to be included separately but I have moved it into a test group now as per https://github.com/datagovuk/find_data_beta/blob/27392939d4b8a2a3b668003e19dc39b5697c1be4/Gemfile#L55